src/layout/sidebar/components/SidebarMenuButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { FC } from "react";
+import styles from "./SidebarMenuButton.module.css";
+
+interface Props {
```

Comment:
Let's name interfaces more meaningfully and starting with an `I` => `ISideMenuButtonProps`